### PR TITLE
Install certificates

### PR DESCRIPTION
**Pull request by phozzy**
To solve #3 issue.

### CODE REVIEW

File: packer/bare-bones.scm [hunk 1]
```diff
@@ -32,13 +33,13 @@
 root ALL=(ALL) ALL
 vagrant ALL=(ALL) NOPASSWD: ALL\n"))
 
-  (packages (filter (lambda (p)
+  (packages (append (list nss-cers) (filter (lambda (p)
```
Comment (palfrey) [inline packer/bare-bones.scm:36]: Thought this was probably the case, and now this project has CI to prove it :) `nss-cers` should be `nss-certs`

See CI log with
```
==> virtualbox-iso: /mnt/etc/config.scm:36:12: error: nss-cers: unbound variable
==> virtualbox-iso: hint: Did you forget a `use-modules' form?
```

Comment (phozzy) [reply packer/bare-bones.scm:36]: I see... I'll check it once I get access to a PC with Linux.

Comment (palfrey) [reply packer/bare-bones.scm:36]: Any luck with this?

Comment (phozzy) [reply packer/bare-bones.scm]: Yep, fixed.

Comment (phozzy) [reply packer/bare-bones.scm]: I think it fails on a later step. During the local build I see that it builds an image, but fails to connect via ssh during a following step.

Comment (palfrey) [reply packer/bare-bones.scm]: Agreed. Something is wrong with the build and I'm trying to fix it in https://github.com/palfrey/guix-vm/pull/5

**Comment by palfrey** [comment]
I haven't been able to fix the build issues, so I've just tested this locally and disabled the Github actions build in #5. I've also filed https://github.com/palfrey/guix-vm/issues/6 around fixing the build issues properly.

**Pull request merged**